docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  redis:
```

Comment:
Nooooo - `docker-compose` is strictly for local development.
In production - we'll use a Redis AWS elasticache instance.

We have a few already running, they're usually named `shared-[ENVIRONMENT-NAME]`.
I think we can safely use those, and not need to spin up a new one because [this line of boot.rb](https://github.com/NYPL-discovery/scsb_item_updater/blob/factor-out-queue-work-SCC-334/boot.rb#L40) namespaces our Redis keys so they don't collide with other apps'.

@kfriedman: Sound right? We can use those `shared-*` Redis instances?

Reply:
Ah, okay — that makes sense, and was more what I was expecting, tbh. My misunderstanding.

Reply:
Maybe I can make that clearer in the wording and formatting of the README.